front/style/pages/_grande-marche-europe.scss

```diff
@@ -0,0 +1,201 @@
+.gme {
+    overflow: hidden;
+
+    .splash-screen {
+        @include splash--screen( $blue--europe, 2s, 300);
+
+        .logo {
+            @include vertical--align;
+            z-index: 400;
+            position: relative;
+            font-size: $text--size-extra-large;
+            @include text--bold;
+            opacity: 0;
+            transition: all 1s;
+
+            img {
+                width: 30%;
+
+                @media (max-width: $mobile) {
+                    width: 50%;
+                }
+            }
+        }
+    }
+
+    .filter {
+        background-color: $blue--europe;
+        height: 101%;
```

Comment:
`101%` weird, no?

Reply:
Yes ... 
It was a fix, but i just clean it.